browser-test/src/number_input.test.ts

```diff
@@ -1,9 +1,27 @@
 import {
-  startSession, loginAsAdmin, loginAsTestUser, logout, selectApplicantLanguage, AdminQuestions, AdminPrograms, ApplicantQuestions, endSession
+  startSession, loginAsAdmin, loginAsTestUser, logout, selectApplicantLanguage, AdminQuestions, AdminPrograms, ApplicantQuestions, waitForPageJsLoad, endSession
 } from './support'
 
+describe('input type', () => {
+  it('sets the inputmode attribute to decimal', async () => {

```

Comment:
I don't think we need this test as it is more of a "change detector" test (Did the attributes change?) where we should be testing functionality as you have below.

Reply:
removed this and added a test to `FieldWithLabelTest.java` to test for this